pkg/specgen/specgen.go

```diff
@@ -384,6 +384,10 @@ type ContainerSecurityConfig struct {
 	// ReadOnlyFilesystem indicates that everything will be mounted
 	// as read-only
 	ReadOnlyFilesystem bool `json:"read_only_filesystem,omitempty"`
+	// ReadWriteTmpfs indicates that when running with a ReadOnlyFilesystem
+	// mount temporary file systems
+	ReadWriteTmpfs bool `json:"read_wrie_tmpfs,omitempty"`
```

Comment:
Typo in the json name, should be `read_write_tmpfs`

Reply:
Yikes.